smack-experimental/src/main/java/org/jivesoftware/smackx/mam/MamManager.java

```diff
@@ -289,10 +320,11 @@ private static void addStart(Date start, DataForm dataForm) {
      * @throws InterruptedException
      * @throws NotLoggedInException
      */
-    public MamQueryResult page(DataForm dataForm, RSMSet rsmSet) throws NoResponseException, XMPPErrorException,
+    public MamQueryResult page(Jid objectJID, String node, DataForm dataForm, RSMSet rsmSet) throws NoResponseException, XMPPErrorException,
```

Comment:
Please also keep the `page(DataForm, RSMSet)` method. Smack's 4.2 API is frozen, if this shall go into 4.2 you must not break the API.